custom/tkp/solver.h

```diff
@@ -0,0 +1,13 @@
+#include <bits/stdc++.h>
+
+struct Event {
+  int x, y;
+  int capacity;
+};
+
+// Forward declaration.
```

Comment:
- [ ] remove comment, outdated after separating initial main into main and solver.